superset/assets/stylesheets/less/variables.less

```diff
@@ -0,0 +1,117 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/************************************************************************/
+/* COLORS                                                               */
+/* Please attempt to use and standardize on these colors,               */
+/* rather than including specific color values in                       */
+/* component styles. This will allow us to more easily adjust theming   */
+/************************************************************************/
+
+@indicator-color: #44C0FF;
+
+@almost-black: #263238;
+@gray-dark: #484848;
+@gray: #879399;
+@gray-light: #CFD8DC;
+@gray-bg: #f5f5f5;
+@gray-heading: #A3A3A3;
+@menu-hover: #F2F3F5;
+@lightest: #ffffff;
+@darkest: #000000;
+
+/* toasts */
+@pink: #E32364;
+@purple: #2C2261;
```

Comment:
It would be nice to have colors named by their role in the UI, rather than the default color they are set to, for better custom theming. Some of them are already named this way, like `indicator-color` and `menu-hover`.

This might be outside the scope of this PR, because defining parameters for a theme is pretty involved.